libdrgn/language.h

```diff
@@ -155,6 +156,21 @@ drgn_element_format_object_flags(enum drgn_format_object_flags flags)
 		(flags & DRGN_FORMAT_OBJECT_ELEMENT_TYPE_NAMES) >> 2);
 }
 
+/** Return the drgn language spec corresponding to the given dwfl language */
+static const struct drgn_language *drgn_dwarf_to_language(int dwfl_lang)
+{
+	switch (dwfl_lang) {
+	case DW_LANG_C:
+	case DW_LANG_C89:
+	case DW_LANG_C99:
+	case DW_LANG_C11:
+		return &drgn_language_c;
+	// TODO Provide proper fallback errors instead of defaulting to C always.
```

Comment:
I don't think we'll ever want to make this fail hard. I'd rather have a partially functional debugger for an unknown language than one that doesn't start up at all. Long term, we probably want a `drgn_language_unknown` that has sane default operations, but for now C is fine.